doc/Writerside/topics/validation.md

```
@@ -0,0 +1,34 @@
+: Opens a file.
+
+-c, --close
+: Closes a file.
+
+-v, --version
+: Displays version information.
+
+-h, --help
+: Displays help.
+
+<seealso>
```

Markdownlint

Element: seealso
Inline HTML

(MD033, no-inline-html)